docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -8,12 +8,11 @@ The Cassandra connector allows querying data stored in
 Requirements
 ------------
 
-Requirements for using the connector in a catalog to connect to a Cassandra
-data source are:
+To connect to an Cassandra data source, you need:
```

Comment:
I am not sure about this simple entry sentence since it leaves out the fact that you use the connector in a catalog. Maybe its fine to leave out though. Wdyt @electrum ?

Reply:
Alternative:

> To connect to a Cassandra data source using the connector in a catalog, you need:

Reply:
I like that better.. maybe sync with @electrum asap so this is updated and merged before the release is cut

Reply:
@electrum let me know if you like the suggestion to include connector/catalog in the sentence. Thanks. All other updates were applied.